flow-sample/src/main/java/com/example/flow/view/TabletMasterDetailRoot.java

```diff
@@ -66,6 +66,7 @@ public TabletMasterDetailRoot(Context context, AttributeSet attrs) {
         if (countDown == 0) {
           disabled = false;
           wrapped.onTraversalCompleted();
+          ((IsMasterView) masterContainer.getCurrentChild()).setSingleChoiceMode();
```

Comment:
Thanks for the patch!

I'd feel fine with just doing it in both updateSelection() methods. Not sure the DRY violation is any uglier, and it really is a view concern.

Have you signed our contributor agreement? https://github.com/square/flow/blob/master/CONTRIBUTING.md